memory/ndctl.py

```diff
@@ -322,6 +339,23 @@ def test_namespace(self):
         namespaces = self.get_json('-N')
         self.log.info('Created namespace %s', namespaces)
 
+    def test_disable_enable_ns(self):
+        """
+        Test enable disable namespace
+        """
+        region = self.get_default_region()
+        for _ in range(0, 3):
```

Comment:
why loop ?

Reply:
to execute multiple times

Reply:
:) yes i understand that but the question is test requires to create three namespaces ?

Reply:
yes to enable/disable namespaces it has to be created first :)